src/biogeochem/DustEmisFactory.F90

```diff
@@ -25,33 +25,41 @@ function create_dust_emissions(bounds, NLFilename) result(dust_emis)
         !---------------------------------------------------------------------------
         ! Create a dust_emission base class objecct
         ! The method implemented depends on namelist input
+        !
+        ! DESIGN NOTES:   Erik Kluzek 07/15/2024
```

Comment:
@adrifoster if you can I'd be interested in your feedback on this little design note section I added. Thanks...

Reply:
Good note. I like this strategy better (at least in this instance) since as you say we need both CAM and CTSM to know about the dust emission type. It's better to have it chosen as a function that we can control elsewhere. CTSM doesn't need to know anything about *how* we know which dust emission it is, we just need to know is it this? is it that?

Reply:
Awesome. Thanks for looking at it.

Notes like this might be something appropriate to do in other places when we bring in new code and we want to talk about the design. I think it is better kept here in the code for this case than under the doc/design directory.